src/linear_algebra/eigenspace.lean

```diff
@@ -80,10 +81,24 @@ lemma mem_eigenspace_iff {f : End R M} {μ : R} {x : M} : x ∈ eigenspace f μ
 by rw [eigenspace, linear_map.mem_ker, linear_map.sub_apply, algebra_map_End_apply,
   sub_eq_zero]
 
+lemma has_eigenvector.apply_eq_smul {f : End R M} {μ : R} {x : M} (hx : f.has_eigenvector μ x) :
```

Comment:
```suggestion
@[simp] lemma has_eigenvector.apply_eq_smul {f : End R M} {μ : R} {x : M} (hx : f.has_eigenvector μ x) :
```

Reply:
Is this something `simp` can do?  I don't think it can check side conditions?

Reply:
Oops you're right; let's drop this suggestion.